src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateHoldingEventHandler.java

```diff
@@ -118,6 +121,9 @@ public CreateHoldingEventHandler(Storage storage, MappingMetadataCache mappingMe
               future.complete(dataImportEventPayload);
             })
             .onFailure(e -> {
```

Comment:
Did you think on approach without modifying all handlers, instead of matching error message on data-import-processing-core and skip sending DI_ERROR? Or you think this approach that in PR more suitable?